README.md

```diff
@@ -318,9 +318,15 @@ For more information:
 Load the module:
 
 ```lua
-local vscode = require('vscode-neovim')
+local vscode = require('vscode')
 ```
 
+<!-- prettier-ignore-start -->
+> [!TIP]
+> "vscode" is an alias for "vscode-neovim" module.
+> So don't be confused if "vscode-neovim" is used in the below examples.
```

Comment:
Just change the examples. We can change all wording/internals to use `vscode` and have `vscode-neovim` as an undocumented backwards-compatibility.